### PR TITLE
Create space modals

### PR DESCRIPTION
**Pull request by abhishek-01k**
Modals has been generated for spaces. 

-> Note: Schedule Date and time modal is not completed because of lack of date time picker package. Will update it later on.

### CODE REVIEW

**Comment by deekshithepns** [comment]
better to add hover effect for 'skip' button

**Comment by deekshithepns** [comment]
Screenshot (33)
width of date and time containers need to be of same width

**Comment by deekshithepns** [comment]
Screenshot (34)

fix the color of date picker icon and texts

**Comment by deekshithepns** [comment]
Screenshot (35)
This one I found for mobile screen... closing modal on outside click is not working

File: src/sections/space/SpaceSidebarSection.tsx [hunk 1]
```diff
@@ -6,19 +6,21 @@ import styled,{useTheme} from 'styled-components';
...
 // Internal Components
 import { SpaceSidebar, SpaceSidebarTabs } from 'components/space';
+
+// Internal Configs
 import {ItemVV2 } from 'components/reusables/SharedStylingV2';
 import { SpaceLocalContext } from 'contexts';
 import ProfileHeader from 'components/chat/w2wChat/profile';
 
-export const SpaceSidebarSection = () => {
+export const SpaceSidebarSection = ({showCreateSpaceModal}) => {
```
Comment (mishramonalisha76) [inline src/sections/space/SpaceSidebarSection.tsx:15]: type for props

**Pull request merged**